src/features/PeopleList/index.js

```diff
@@ -1,3 +1,38 @@
+import { PersonTile } from "./PersonTile";
+import { Content, List, Title, Wrapper } from "./styled";
```

Comment:
You dont use here List in import from /styled

Reply:
True, forgot to remove it. I'll do it today